src/pages/office/cat-cpc/office.cat-cpc.page.ts

```diff
@@ -0,0 +1,384 @@
+        select(getJournalData),
+        select(getCandidate),
+        select(getCandidateDriverNumber),
+        map(formatDriverNumber),
+      ),
+      displayCandidateDescription$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getCandidateDescription))),
+        map(([outcome, candidate]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'candidateDescription', candidate)),
+      ),
+      candidateDescription$: currentTest$.pipe(
+        select(getTestSummary),
+        select(getCandidateDescription),
+      ),
+      displayIdentification$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getIdentification))),
+        map(([outcome, identification]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'identification', identification)),
+      ),
+      identification$: currentTest$.pipe(
+        select(getTestSummary),
+        select(getIdentification),
+      ),
+      combination$: currentTest$.pipe(
+        select(getTestData),
+        select(getCombination),
+      ),
+      displayAdditionalInformation$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getAdditionalInformation))),
+        map(([outcome, additional]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'additionalInformation', additional)),
+      ),
+      additionalInformation$: currentTest$.pipe(
+        select(getTestSummary),
+        select(getAdditionalInformation),
+      ),
+      assessmentReport$: currentTest$.pipe(
+        select(getTestSummary),
+        select(getAssessmentReport),
+      ),
+      overallScore$: currentTest$.pipe(
+        select(getTestData),
+        select(getTotalPercent),
+      ),
+      question1$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion1),
+      ),
+      question2$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion2),
+      ),
+      question3$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion3),
+      ),
+      question4$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion4),
+      ),
+      question5$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion5),
+      ),
+      testResult$: currentTest$.pipe(
+        select(getTestOutcomeDebrief),
+      ),
+      isRekey$: currentTest$.pipe(
+        select(getRekeyIndicator),
+        select(isRekey),
+      ),
+    };
+
+    const { testResult$, combination$ } = this.pageState;
+
+    this.subscription = merge(
+      testResult$.pipe(map(result => this.outcome = result as TestOutcome)),
+      combination$.pipe(map(result => this.combinationAdditionalText = result as CombinationCodes)),
```

Comment:
the name 'combinationAdditionalText' is misleading, should be renamed to something like 'combinationCode'

Reply:
its the additional text from the combination, not the code...Ie 'Fire ex'

Reply:
its 'LGV1' for example, not the suffix

Reply:
i think i edit'd whilst u were still writing...